src/config.py

```diff
@@ -10,3 +10,4 @@
 GITHUB_OAUTH_TOKEN_SECRET_ARN = os.getenv('GITHUB_OAUTH_TOKEN_SECRET_ARN')
 REGION = os.getenv('AWS_DEFAULT_REGION')
 DELETE_PREVIOUS_COMMENTS = os.getenv('DELETE_PREVIOUS_COMMENTS') == 'true'
+COMMENT_ON_SUCCESS = os.getenv('COMMENT_ON_SUCCESS', 'True')
```

Comment:
```suggestion
COMMENT_ON_SUCCESS = os.getenv('COMMENT_ON_SUCCESS') == 'true'
```

Reply:
@jlhood can you explain what this pattern is doing? As I read it, if `COMMENT_ON_SUCCESS` is not set during the tests (which it is not), then this would be false/null == true which would resolve to falsey? This would the opposite of what I intended the default to be (which is to send a comment on success).

Reply:
@jflasher In the actual app, this env var will always be populated with either the string 'true' or 'false' since the template always passes the parameter value through to the Lambda function env var. For unit tests, conftest.py sets default env vars for all tests. Then we can override config on individual tests. I just pushed a commit to add this new env var to conftest.py which should address the failing unit test.